ml-proto/README.md

```diff
@@ -82,20 +82,20 @@ Note however that the REPL currently is too dumb to allow multi-line input. :)
 See `wasm -h` for (the few) options.
 
 
-## Core Language vs External Language
+## Official Language Specification vs Internal Kernel Language
```

Comment:
How about "Full Language vs Kernel Language"? (I'm not sure how to contrast the "official language specification" with what is described here. Is there another language specification?)